website/src/pages/en/ai-suite/ai-introduction.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: The Graph's AI
+---
+
+Build with speed and scale faster with The Graph's AI Suite.
+
+## What is AI on The Graph?
+
+AI on The Graph connects large language models (LLMs) like Claude to onchain data indexed by The Graph. This allows developers to build AI systems that query blockchain data directly using natural language. This eliminates the need to write code or manage infrastructure.
+
+Instead of relying on static datasets or centralized APIs, you can now access live, verifiable blockchain data using [Graph Assistant](/ai-suite/graph-assistant/introduction/), [Subgraph MCP](), [Token API MCP]().
+
+Fetch fast access to accurate, real-time onchain data.
+
+## Why Use Onchain Data with AI?
+
+If you're building with AI in web3, onchain data is necessary. Wallets, tokens, governance, and transactions all live onchain, but are often very difficult to access at scale without an indexing or custom solution.
+
+The Graph solves this by making onchain data available through [Subgraphs]() and [TokenAPI]() via [Graph Assistant]().
```

Comment:
I'm repeating myself a bit here just to highlight another place where the same thing applies, we actually have two different stories:

1. MCPs -> AI as an interface -> developers can talk with the graph's data from their own agents or build AI-enabled apps on top of TG.
2. Graph Assistant -> AI as a product -> solution for non-technical people to gain insights from TG's data.